deployment/sids-survey-backend-deployment.yml

```diff
@@ -0,0 +1,86 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: sids-survey-backend
+  namespace: ml-app
+spec:
+  selector:
+    app: sids-survey-backend
+  ports:
+  - protocol: TCP
+    port: 80
+    targetPort: 80
+  type: ClusterIP
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: sids-survey-backend
+  namespace: ml-app
+  labels:
+    app: sids-survey-backend
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: sids-survey-backend
+  template:
+    metadata:
+      labels:
+        app: sids-survey-backend
+    spec:
+      containers:
+      - name: sids-survey-backend
+        image: acrmlbackend.azurecr.io/sidmlbackend:latest
+        resources:
+          limits:
+            cpu: 2000m
+            memory: 2048Mi
+          requests:
+            cpu: 100m
+            memory: 512Mi
+        volumeMounts:
+          - name: azure
+            mountPath: /mnt/azure
+        ports:
+        - containerPort: 80
+        env:
+        - name: AzureFunctionsJobHost__functions__0
+          value: main
+        - name: AzureWebJobsSecretStorageType
+          value: kubernetes
+        - name: AzureWebJobsKubernetesSecretName
+          value: secrets/func-keys-kube-secret-sidmlbackend
+        - name: MODEL_SERVICE
+          value: clustering
+        envFrom:
+        - secretRef:
+            name: sidmlbackend
+        readinessProbe:
+          failureThreshold: 3
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 240
+          httpGet:
+            path: /
+            port: 80
+            scheme: HTTP
+        startupProbe:
+          failureThreshold: 3
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 240
+          httpGet:
+            path: /
+            port: 80
+            scheme: HTTP
+      serviceAccountName: sidmlbackend-function-keys-identity-svc-act
+      volumes:
```

Comment:
If you don't have a external dataset, or no need of a shared file storage you can remove the volume part.